% min master timer Poulated

```diff
@@ -225,12 +225,14 @@ export function beforeRender(delta) {
          values = array(pixelCount) //I moved these variables into before render
          hues = array(pixelCount) //
          for (i = 0; i < pixelCount; i++) {
-         values[i] -= .005 * delta * .1
-         if (values[i] <= 0) {
-         values[i] = random(1)
-          hues[i] = time(4.6 / 65.536) + 0.2 * triangle(i / pixelCount)
+          values[i] -= .005 * delta * .1
+          if (values[i] <= 0) {
+            values[i] = random(1)
+            hues[i] = time(4.6 / 65.536) + 0.2 * triangle(i / pixelCount)
+          }
+        }
 
```

Comment:
The two } above this line were what was missing. You needed to close out the for loop and the if block

It should look like:

for (someExpression) {
  // code
}

if (someExpression) {
  // code
}